mkdocs.yaml

```diff
@@ -4,6 +4,7 @@ nav:
   - Introduction: 'index.md'
   - 'Developing Docs': 'https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/3924787306/Developing+Documentation'
   - Components: '*include ./components/*/mkdocs.yml'
+  - Tools:      '*include ./tools/*/mkdocs.yaml'
```

Comment:
This was a bit puzzling because I noticed at the top level we have "mkdocs.yaml", but elsewhere we have "mkdocs.yml". I guess the top-level one should be changed?

Reply:
done